scripts/test/Abins/AbinsCalculateSPowderTest.py

```diff
@@ -52,7 +53,7 @@ def test_invalid_input(self):
             good_data = AbinsData.from_dict(json.load(fp))
 
         # invalid filename
-        with self.assertRaises(ValueError):
+        with self.assertRaises(ValidationError):
```

Comment:
Its better to use `self.assertRaisesRegex()`  (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex)

This allows you to pass in a substring / regex pattern to search for in the error message. This means you can ensure the raised exception is the one your were expecting. We've had cases in the past where `assertRaises` was hiding validation / test bugs because the wrong exception of the same type was being triggered.

Reply:
I don't think ValidationError can come from anywhere but Pydantic... but it doesn't hurt to be specific.

Reply:
Done!